projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -752,16 +765,130 @@ function wpcom_add_new_page_check( $post_id, $post ) {
 }
 add_action( 'wp_insert_post', 'wpcom_add_new_page_check', 10, 3 );
 
+/**
+ * Determine if the site has an 'About' page. We do this by loading the `en` annotation for the site and theme. We then check if there is a post with the title 'About'. If there is, we get the `hs_old_id` and check to make sure a corresponding post exists in the user's locale.
+ *
+ * @return int|null The post ID of the 'About' page if it exists, null otherwise.
+ */
+function wpcom_get_site_about_page_id() {
+	$annotation = wpcom_get_theme_annotation( get_stylesheet() );
```

Comment:
The logic in this function feels like it could be relatively heavy/slow, as we may be reading annotation content from disk, and then performing a database query to find the relevant pages. Might it make sense to cache this result and push the computation into a secondary function? The only edge case we'd need to handle would be situations where we're running this code before Headstart has been run for the site, which should be unlikely. (I think we may also set an option while Headstart is running, so we may be able to skip caching the result in that case.)